psydac/fem/grid.py

```diff
@@ -31,10 +31,10 @@ class FemAssemblyGrid:
         1D finite element space.
 
     start : int
-        Index of first 1D basis local to process.
+        Index of first element local to process.
 
     end : int
-        Index of last 1D basis local to process.
+        Index of last element local to process.
```

Comment:
Question: do we mean `cell` here, or `coefficient`?

Reply:
cell